x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -0,0 +1,73 @@
+[[configuring-stack-security]]
+== Configure security for the Elastic Stack
+++++
+<titleabbrev>Configuring security</titleabbrev>
+++++
+
+Security needs vary depending on whether you're developing locally on your
+laptop or securing all communications in a production environment. Because
+security needs vary, the following scenarios provide different paths for
+configuring the Elastic Stack.
+
+TIP: Each subsequent scenario builds on the previous one so that you can add
+additional security by building on the existing layer.
+
+These scenarios don't cover every situation, but provide a framework for
+securing {es} and the Elastic Stack based on typical use cases.
+
+image::images/elastic-security-overview.png[Elastic Security layers]
+
+[discrete]
+[[security-minimal-overview]]
+=== Minimal security
+
+If you want to set up {es} on your laptop and start developing, this
+scenario is for you. This configuration implements usernames and passwords for
+built-in users to cover the minimal security requirements for running a local
+{es} cluster.
+
+WARNING: Do not deploy this security configuration in production environments.
+All communications are sent as plain text, which is unsafe for production
+environments.
+
+<<security-minimal-setup,Set up minimal security>>
+
+[discrete]
+[[security-basic-overview]]
+=== Basic security
+
+This scenario covers how to configure basic security to prevent unauthorized
+access to your {es} cluster. You'll build on the minimal security
```

Comment:
I'll update this to read:
>This scenario covers how to configure basic security to prevent unauthorized nodes from joining your Elasticsearch cluster.